packages/client/src/service/txpool.ts

```diff
@@ -457,11 +458,29 @@ export class TxPool {
 
       // Broadcast to peer if at least 1 new tx hash to announce
       if (hashesToSend.length > 0) {
-        try {
-          await peer.eth?.request('NewPooledTransactionHashes', hashesToSend)
-        } catch (e) {
-          this.markFailedSends(peer, hashesToSend, e as Error)
-        }
+        if (
+          peer.eth !== undefined &&
+          peer.eth['versions'] !== undefined &&
+          peer.eth['versions'].includes(68)
```

Comment:
Ah ok the `eth/66`/`eth/68` switch is handled here.

Reply:
Yup, I'll add a comment to clarify what's going on